programs/lfx-mentorship/2023/02-Jun-Aug/README.md

```diff
@@ -28,25 +28,64 @@ Mentee application instructions can be found on the [Program Guidelines](https:/
 
 Table of Contents
 
-* [Armada](#armada)
-* [CoreDNS](#coredns)
-* [Jaeger](#jaeger)
-* [Knative](#knative)
-* [Kubescape](#kubescape)
-* [Kyverno](#kyverno)
-* [Notary](#notary)
-* [Meshery](#meshery)
-* [ORAS](#oras)
-* [Tetragon](#tetragon)
-* [WasmEdge](#wasmedge)
-* [Konveyor](#konveyor)
-* [Strimzi](#strimzi)
-* [Thanos](#thanos)
-* [KubeArmor](#kubearmor)
-* [LitmusChaos](#litmuschaos)
-* [Vitess](#vitess)
-* [KubeVela](#kubevela)
-* [Service Mesh Performance](#service-mesh-performance)
+- [Armada](#armada)
+  - [Build interfaces around Postgres for Armada](#build-interfaces-around-postgres-for-armada)
+- [Cilium/Tetragon](#ciliumtetragon)
+  - [Implement a Kubernetes operator to maintain pod IP to pod metadata mapping](#implement-a-kubernetes-operator-to-maintain-pod-ip-to-pod-metadata-mapping)
+- [CNCF Landscape ](#cncf-landscape)
+  - [UX / UI Improvements II](#ux--ui-improvements-ii)
+- [CoreDNS](#coredns)
+  - [Add DNS-over-QUIC (DoQ) and/or DNS-over-HTTP/3 (DoH3) support](#add-dns-over-quic-doq-andor-dns-over-http3-doh3-support)
+- [Jaeger](#jaeger)
+  - [Upgrade Jaeger's internal telemetry to OpenTelemetry](#upgrade-jaegers-internal-telemetry-to-opentelemetry)
+  - [Implement Critical Path analysis](#implement-critical-path-analysis)
+- [Konveyor](#konveyor)
+  - [Add Integration test suite and components testing to Konveyor](#add-integration-test-suite-and-components-testing-to-konveyor)
+- [Knative](#knative)
+  - [Self-Balancing Knative Kafka Broker partitions](#self-balancing-knative-kafka-broker-partitions)
+  - [Porting Knative Serving to Microshift](#porting-knative-serving-to-microshift)
```

Comment:
Please swop these two

Reply:
@DewaldtKleinhans thankyou for noticing it.